src/components/containerGradient/saber/index.module.scss

```diff
@@ -0,0 +1,82 @@
+@mixin saber-background($position) {
+  content: '';
+  position: absolute;
+  z-index: -1;
+  #{$position}: 0;
+
+  @if ($position == left or $position == right) {
+    top: 0;
+    height: 100%;
+    width: 100px;
+  } @else {
+    width: 100%;
+    left: 0;
+    height: 50px;
+  }
+
+  background: linear-gradient(
+    /* vise versa */ to #{$position},
+    rgb(0, 0, 0) 0%,
+    rgba(var(--color-r), var(--color-g), var(--color-b), 0.11) 100%
+  );
+}
+
+// check Storybook for more documentation
+$valid-colors: 'green', 'blue', 'purple', 'white', 'red', 'yellow';
+$valid-positions: 'left', 'right', 'top', 'bottom';
+$saber-border-width: 2px;
+
+@mixin saber($color: 'green', $position) {
+  position: relative;
+  border-#{$position}: $saber-border-width solid white;
+
+  @if ($color == 'green') {
+    --color-r: 54;
```

Comment:
Not understand why like this?
just make 
$green =  rgba() - GLOBAL variables and reuse that everywhere

Reply:
will not work, will using later with opasity
rbga(r, b,g, opasity)

Reply:
why rgba not work with opacity? O_o didn't catch your tought

Reply:
it will not work like this, so not possible to write vars better. I would want too
<img width="347" alt="image" src="https://github.com/cybercongress/cyb/assets/18665326/e7189209-79c9-462d-936f-c1d45d978709">